src/lib/core/partialcubeop.ts

```diff
@@ -0,0 +1,45 @@
+import { CubeIndex } from "lib/core/cubeindex";
```

Comment:
nit: since this is in the same folder, should it be a relative path? Not sure what guidelines we want to follow there

Reply:
I thought we wanted all paths to be absolute?

Reply:
yeah I prefer absolute pathing here

I think the only times I would use a relative path would be for tests and the scss/non-ts files